plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1219,13 +1239,25 @@ public void addColumn(ConnectorSession session, ConnectorTableHandle tableHandle
         try {
             long commitVersion = handle.getReadVersion() + 1;
 
+            AtomicInteger maxColumnId;
+            switch (columnMappingMode) {
+                case NONE -> {
```

Comment:
Is the wrapping intentional or rather a left-over?

Reply:
It's intentional. Otherwise, it throws "InnerAssignment: Inner assignments should be avoided."